src/main/java/stream/flarebot/flarebot/commands/general/PollCommand.java

```diff
@@ -1,19 +1,353 @@
 package stream.flarebot.flarebot.commands.general;
 
+import net.dv8tion.jda.core.EmbedBuilder;
 import net.dv8tion.jda.core.entities.Member;
 import net.dv8tion.jda.core.entities.Message;
 import net.dv8tion.jda.core.entities.TextChannel;
 import net.dv8tion.jda.core.entities.User;
+import org.apache.commons.lang3.text.WordUtils;
+import stream.flarebot.flarebot.FlareBot;
 import stream.flarebot.flarebot.commands.Command;
 import stream.flarebot.flarebot.commands.CommandType;
 import stream.flarebot.flarebot.objects.GuildWrapper;
+import stream.flarebot.flarebot.objects.Poll;
+import stream.flarebot.flarebot.objects.PollOption;
+import stream.flarebot.flarebot.util.ColorEnum;
+import stream.flarebot.flarebot.util.GeneralUtils;
 import stream.flarebot.flarebot.util.MessageUtils;
 
+import java.awt.Color;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.stream.Collectors;
+
 public class PollCommand implements Command {
 
     @Override
     public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Message message, String[] args, Member member) {
-        MessageUtils.sendErrorMessage("This command is currently disabled while it is being redone!", channel);
+        if (args.length == 0) {
+            int closedSize = guild.getPolls().stream().filter(Poll::isClosed).collect(Collectors.toList()).size();
+            if (guild.getPolls().size() == 0 || closedSize > 0) {
```

Comment:
isEmpty ans how come you are doing or closed > 0? I could have 2 open and 1 closed and this would fire